src/Common/src/System/Net/HttpKnownHeaderNames.cs

```diff
@@ -68,5 +70,7 @@ internal static class HttpKnownHeaderNames
         public const string Warning = "Warning";
         public const string XAspNetVersion = "X-AspNet-Version";
         public const string XPoweredBy = "X-Powered-By";
+
+        // When adding a new constant, add it to HttpKnownHeaderNames.TryGetHeaderName.cs as well.
```

Comment:
Yeah, I intentionally added the same comment at both the top and bottom of the file. Based on your feedback, I went ahead and removed the comment at the bottom. One comment at the top should be sufficient.